src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml

```diff
@@ -36,107 +37,149 @@
                     </DataTrigger>
                 </Style.Triggers>
             </Style>
+
+            <Style x:Key="OverlayTextBlockStyle" TargetType="TextBlock" BasedOn="{StaticResource CellTextBlockStyle}">
+                <Setter Property="TextAlignment" Value="Center"/>
+                <Setter Property="HorizontalAlignment" Value="Center"/>
+                <Setter Property="Foreground" Value="{DynamicResource {x:Static vsShell:VsBrushes.InactiveCaptionTextKey}}" />
+            </Style>
+
+            <Style x:Key="OverlayTitleTextStyle" TargetType="TextBlock" BasedOn="{StaticResource OverlayTextBlockStyle}">
+                <Setter Property="FontWeight" Value="Bold"/>
+            </Style>
+
+            <Style x:Key="OverlayGridStyle" TargetType="Grid">
+                <Setter Property="Background" Value="{DynamicResource {x:Static vsShell:VsBrushes.BrandedUIBackgroundKey}}" />
+                <Setter Property="Visibility" Value="Collapsed" />
+                <Style.Triggers>
+                    <DataTrigger Binding="{Binding HasServerIssues}" Value="False">
+                        <Setter Property="Grid.Visibility" Value="Visible"/>
+                    </DataTrigger>
+                </Style.Triggers>
+            </Style>
         </ResourceDictionary>
 
     </UserControl.Resources>
-    
     <Grid UseLayoutRounding="True">
-        <Grid.ColumnDefinitions>
-            <ColumnDefinition Name="Table" Width="*"/>
-        </Grid.ColumnDefinitions>
-        <DataGrid Grid.Column="0" 
+
+        <Grid x:Name="Overlay" Style="{StaticResource OverlayGridStyle}" Panel.ZIndex="1000">
```

Comment:
As we discussed offline, this should be a more generalized message.
Maybe something like:

```
No Taint Vulnerabilities were found.

Please ensure that your project is bound to a Sonar project that contains taint vulnerabilities.

Click here to go online and find out more about this and other SonarLint features.
```